utils/src/server/rpc_module.rs

```diff
@@ -147,7 +146,8 @@ impl RpcModule {
 						sub_id,
 						is_online: online_tx,
 					};
-					callback(params, sink)
+					// TODO: maybe params, context, sink is a more natural order?
```

Comment:
Any strong opinions on this?

Reply:
no, //cc @maciejhirsz

Reply:
are planning to fix this in the current PR? Otherwise, open an issue and link to it

Reply:
We can revisit if it becomes annoying in practise, no need to decide now.